pkg/cmd/redo/apply.go

```diff
@@ -38,6 +41,25 @@ func (o *applyRedoOptions) addFlags(cmd *cobra.Command) {
 	cmd.MarkFlagRequired("sink-uri") //nolint:errcheck
 }
 
+func (o *applyRedoOptions) complete(cmd *cobra.Command) error {
+	// parse sinkURI as a URI
+	sinkURI, err := url.Parse(o.sinkURI)
+	if err != nil {
+		return cerror.WrapError(cerror.ErrSinkURIInvalid, err)
+	}
+	rawQuery := sinkURI.Query()
+	if rawQuery.Get("safe-mode") != "true" {
+		cmd.Println("Warning: safe-mode is not in sink-uri, " +
```

Comment:
This warning is not accurate if `safe-mode=false` is in sink-uri. Should it be replaced with `Warning: safe-mode is not in sink-uri or the value is false` ?